test/nn/test_multihead_attention.py

```diff
@@ -589,7 +589,8 @@ def test_multihead_self_attn_two_masks_fast_path_mock(self, device):
         Multihead self-attention should take fast path when both attention mask (mask type 0)
         and key padding mask (mask type 1) are provided at the same time on CPU and CUDA
         """
-        if device not in ['cpu', 'cuda']:
+        device = device.rstrip(':0123456789')
+        if device not in ['cpu', 'cuda', torch._C._get_privateuse1_backend_name()]:
```

Comment:
nit: update skipTest below with 
`self.skipTest("Fastpath only runs on CPU and CUDA and PrivateUse1.")`

Reply:
> nit: update skipTest below with `self.skipTest("Fastpath only runs on CPU and CUDA and PrivateUse1.")`

updated